docs/specification/current.md

```diff
@@ -913,7 +913,7 @@ iat | REQUIRED | *number* | The time at which this JWT was issued, expressed in
 jti | REQUIRED | *string* | A nonce string value that uniquely identifies this authentication JWT (used to protect against replay attacks).
 tenant | OPTIONAL | *string* | An opaque string identifying the healthcare organization that is invoking the CDS Hooks request.
 
-CDS Services SHOULD maintain an allowlist of the `iss` and `jku` fields to only the CDS Clients they trust.
+CDS Services SHOULD maintain an allowlist of the `iss` and `jku` fields containing only the CDS Clients they trust.
```

Comment:
```suggestion
CDS Services SHOULD limit the CDS Clients they trust by maintaining an allowlist of `iss` and `jku` urls.
```

Agree that initial wording was awkward. It was trying to indicate that iss and jku urls pointed to CDS Clients. Your proposed wording is also awkward in that an allowlist doesn't contain CDS Clients, rather, their iss/jku's.